src/Workspaces/SharedUtilitiesAndExtensions/Workspace/Core/Extensions/ISolutionExtensions.cs

```diff
@@ -55,9 +55,9 @@ public static Document GetRequiredDocument(this Solution solution, DocumentId do
             return document;
         }
 
-        public static Document GetRequiredDocument(this Project project, DocumentId documentId)
+        public static TextDocument GetRequiredTextDocument(this Solution solution, DocumentId documentId)
```

Comment:
I think this is used in a bunch of other places, not sure we should be changing it
http://sourceroslyn.io/#Microsoft.CodeAnalysis.Workspaces/ISolutionExtensions.cs,d56b2beb86c3b187,references

Reply:
Again, I just moved it since it is a Project extension and therefore really belongs in ProjectExtensions.

Reply:
@mgoertz-msft ah I see - they'll use the project extensions one.  Ok should be fine to leave as-is (same applies to other comments)